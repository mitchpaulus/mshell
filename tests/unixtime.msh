2025-07-10 orig!
@orig toUnixTime u!
@u str wl
@u fromUnixTime d!
@d "2006-01-02 15:04:05" dateFmt wl
@orig @d = str wl
