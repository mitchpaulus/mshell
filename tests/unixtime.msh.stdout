1752105600
2025-07-10 00:00:00
true
